pkg/core/xds/sockets.go

```diff
@@ -22,8 +22,8 @@ func MeshMetricsDynamicConfigurationSocketName(workdir string) string {
 	return socketName(filepath.Join(workdir, "kuma-mesh-metric-config"))
 }
 
-func OpenTelemetrySocketName(workdir string) string {
-	return socketName(filepath.Join(workdir, "kuma-open-telemetry"))
+func OpenTelemetrySocketName(workdir string, backendName string) string {
+	return socketName(filepath.Join(workdir, "kuma-otel-"+backendName))
```

Comment:
Probably this needs better naming, looking for suggestions

Reply:
maybe but I don't think it's really that important 🤷